### PR TITLE
Compile only benchmarks: helloWorld and videoplayer

### PR DESCRIPTION
**Pull request by LepilkinaElena**

### CODE REVIEW

File: performance/gradle/benchmark.gradle [hunk 1]
```diff
@@ -96,7 +96,7 @@ task konanJsonReport {
                                                   'compilerVersion': "${konanVersion}".toString(),
                                                   'flags': kotlin.targets.native.compilations.main.extraOpts.collect{ "\"$it\"" },
                                                   'benchmarks': benchContents,
-                                                  'compileTime': nativeCompileTime,
+                                                  'compileTime': [nativeCompileTime],
```
Comment (olonho) [inline performance/gradle/benchmark.gradle]: Why list? What else shall be here?

File: performance/videoplayer/build.gradle [hunk 2]
```diff
@@ -0,0 +1,68 @@
+def dist = findProperty('org.jetbrains.kotlin.native.home') ?: 'dist'
+
+def linkerOpts = []
```
Comment (olonho) [inline performance/videoplayer/build.gradle]: Hmm, this looks like not very awesome idea. Isn't it possible to use composite build with the build in `samples`?

File: performance/gradle/benchmark.gradle (hunk 1 above)
Comment (LepilkinaElena) [reply performance/gradle/benchmark.gradle]: There can be several measurements of compile time to get more stable result.

File: performance/videoplayer/build.gradle (hunk 2 above)
Comment (LepilkinaElena) [reply performance/videoplayer/build.gradle]: Then we have no opportunity 

1) to measure clean way without gradle

2) measure several times

3) use default arguments without -opt, -debug

**Pull request merged**